prog/phonons/masses.F90

```diff
@@ -0,0 +1,205 @@
+module dftbp_masses
+  use dftbp_accuracy
+  implicit none
+  private
+
+  public getAtomicMass 
+
+  contains 
+
+  !!* Returns the atomic mass in a.m.u.
+  !!* specie is a character string 
+  function getAtomicMass(specie) result(mass)
```

Comment:
I'd suggest an interface like https://github.com/awvwgk/dftbplus/blob/atom-masses/prog/dftb%2B/lib_common/atomicmass.F90 for this purpose. This easier to maintain and also covers more cases (lowercase symbols for example).